README.md

```diff
@@ -111,6 +111,10 @@ I’m glad you’re interested in CacheAdvance, and I’d love to see where you
 
 Thanks, and happy caching!
 
+## Developing
+
+Run `swift package generate-xcodeproj` from the root of the repository to generate an Xcode project.
```

Comment:
I wouldn’t recommend using the manually generated project if we don’t have to. Opening the `Package.swift` is magical, and it live-reloads. Since our minimum Xcode support is Xcode 11, I’m thinking we should suggest the following:
```suggestion
Double-click on `Package.swift` in the root of the repository to open the project in Xcode.
```

Reply:
TIL. I didn't know you could do that.

Reply:
Now you see why I’m convinced Swift Package Manager is the future 🙂

Reply:
Indeed

Reply:
This is cool! I didn't know about that either!